pages/common/skaffold.md

```diff
@@ -0,0 +1,24 @@
+# skaffold
+
+> A tool that facilitates continuous development for Kubernetes applications.
+> Homepage: <https://skaffold.dev>.
+
+- Build the artifacts:
+
+`skaffold build`
+
+- Build and deploy your app every time your code changes:
+
+`skaffold dev`
+
+- Run a pipeline file:
+
+`skaffold run`
+
+- Run a diagnostic on Skaffold:
```

Comment:
How does it know which diagnostic to run?

Reply:
I have added the default filename for better understanding.